sdcm/cluster.py

```diff
@@ -2788,15 +2788,28 @@ def kill_gemini_thread(self):
                 if kill_result.exit_status != 0:
                     self.log.error('Terminate gemini on node %s:\n%s', loader, kill_result)
 
-    def get_non_system_ks_cf_list(self, loader_node, db_node_ip):
+    def wrapped_get_non_system_ks_cf_list(self, db_node, request_timeout=300):
```

Comment:
then use the `get_non_system_ks_cf_list` in the nemesis

Reply:
I did it for simplify using of get_non_system_ks_cf_list. Otherwise every time when this function need be called, we'll need find the way to select loader

Reply:
removed